File: src/main/java/com/damianryan/octopus/model/DualFuelSingleRate.java
package com.damianryan.octopus.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AccessLevel;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

@Data
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class DualFuelSingleRate {

    @JsonProperty("electricity_standard")
    int electricityStandard;

    @JsonProperty("gas_standard")
    int gasStandard;
}

File: src/main/kotlin/com/damianryan/octopus/model/DualFuelSingleRate.kt
package com.damianryan.octopus.model

import com.fasterxml.jackson.annotation.JsonProperty

data class DualFuelSingleRate(
    @JsonProperty("electricity_standard") val electricityStandard: Int = 0,
    @JsonProperty("gas_standard") val gasStandard: Int = 0,
)